pages/search/[searchProjectType].vue

```diff
@@ -312,8 +312,14 @@
           role="list"
           aria-label="Search results"
         >
+          <div v-if="searchLoading && results?.hits?.length === 0" class="orbit-spinner">
+            <div class="orbit"></div>
+            <div class="orbit"></div>
+            <div class="orbit"></div>
+          </div>
```

Comment:
Could we use the old Modrinth animated logo animation here? this looks a little out of place